tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Session.java

```diff
@@ -312,21 +421,13 @@ public Runner setOptions(RunOptions options) {
      * Execute the graph fragments necessary to compute all requested fetches.
      *
      * <p><b>WARNING:</b> The caller assumes ownership of all returned {@link Tensor Tensors}, i.e.,
-     * the caller must call {@link Tensor#close} on all elements of the returned list to free up
+     * the caller must call {@link Tensor#close} on all returned tensors or {@link Result#close()} to free up
```

Comment:
Do we have guards on the tensors to prevent double closes if the user messes this up?

Reply:
Not yet, I'll have to add a `closed` field to Tensor (which is probably a good idea anyways).